Configuration/StandardSequences/python/Eras.py

```diff
@@ -65,10 +61,8 @@ def __init__(self):
         # Run3 includes the GE1/1 upgrade
         self.Run3 = cms.ModifierChain( self.Run2_2017,self.run3_GEM )
         # Phase2 is everything for the 2023 (2026?) detector that works so far in this release.
-        # include phase 1 stuff until phase 2 tracking is fully defined....
-        self.Phase2 = cms.ModifierChain( self.phase2_common, self.phase2_tracker, self.trackingPhase2PU140, self.phase2_hgcal, self.phase2_muon, self.run3_GEM )
-        # Phase2dev is everything for the 2023 (2026?) detector that is still in development.
-        self.Phase2dev = cms.ModifierChain( self.Phase2, self.phase2dev_common, self.phase2dev_tracker, self.phase2dev_hgcal, self.phase2dev_muon )
+        self.Phase2LReco = cms.ModifierChain( self.phase2_common, self.phase2_hcal, self.phase2_tracker, self.trackingPhase2PU140, self.phase2_hgcal, self.phase2_muon, self.run3_GEM )
+        self.Phase2GReco = cms.ModifierChain( self.phase2_common, self.phase2_tracker, self.trackingPhase2PU140, self.phase2_muon, self.run3_GEM ) # run 2 calorimeters
```

Comment:
GReco means "the subdetectors that are ready for full reco." Right now, we need the run1 calorimeters for this. I expect this situation to persist for several more months at least.